pkgs/tools/system/xmousepasteblock/default.nix

```diff
@@ -0,0 +1,28 @@
+{ xorg, stdenv, libev, fetchFromGitHub, pkgconfig }:
+
+stdenv.mkDerivation {
+  name = "XMousePasteBlock-1.0";
+  src = fetchFromGitHub {
+    owner = "milaq";
+    repo = "XMousePasteBlock";
+    sha256 = "0vidckfp277cg2gsww8a8q5b18m10iy4ppyp2qipr89771nrcmns";
+    rev = "ef7887d75a38628b7c7990c426297beeb639b706";
+  };
+  patchPhase = ''
```

Comment:
Actually,  
```
makeFlags = "PREFIX=$(out)";
```
would be even better (if it works :/).

Reply:
this works :)